ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -28,70 +28,57 @@ Verify User Can Create, Run and Delete A DS Pipeline From DS Project Details Pag
     ...                DS Project UI
     [Tags]    Smoke
     ...       ODS-2206    ODS-2226    ODS-2633
-
```

Comment:
I know that this is more about the personal preferences, but I kind of liked the way this code is split into smaller sections. It's a bit more readable for me. Since this PR isn't about code-style refactoring, do you mind to put these deleted lines back? :stuck_out_tongue_closed_eyes:

Reply:
The line brakes within a KW/Test is not in our code style across 99% of our RF code + for me it looks confusing. @bdattoma what do you think ?

Reply:
I think there is not explicit mention to blank lines withing keywords or test cases. 

For some test cases, having blank lines highlights that it could make sense to split into multiple TCs. If that's not the case, i don't personally have a strong opinion about that.
For keywords, I don't have a strong opinion, but sometime I see that long kw/tc in RF might be difficult to read especially when there are many args. 

I